examples/rbd/plugin-deploy.sh

```diff
@@ -1,15 +1,29 @@
 #!/bin/bash
 
 deployment_base="${1}"
+shift
+kms_base="${1}"
 
-if [[ -z $deployment_base ]]; then
+if [[ -z ${deployment_base} ]]; then
 	deployment_base="../../deploy/rbd/kubernetes"
 fi
 
-cd "$deployment_base" || exit 1
+cd "${deployment_base}" || exit 1
 
 objects=(csi-provisioner-rbac csi-nodeplugin-rbac csi-config-map csi-rbdplugin-provisioner csi-rbdplugin)
 
 for obj in "${objects[@]}"; do
-	kubectl create -f "./$obj.yaml"
+	kubectl create -f "./${obj}.yaml"
+done
+
+if [[ -z ${kms_base} ]]; then
+	kms_base="../../../examples/kms/vault"
+fi
+
+cd "${kms_base}" || exit 1
```

Comment:
use pushd and popd instead of doing cd

Reply:
Thanks... That will definitely be better.